File: week7/getSquares.js
console.log("Works");
var get_squares = function(number_array){
  function small_to_big(a,b) {
    return a - b;
  }

  var perfect_squares_array = [];
  number_array.forEach( function(element) {
    if ( ( Math.sqrt(element) % 1 ) === 0 ){
      if ( !perfect_squares_array.includes(element) ){
        perfect_squares_array.push(element);
      }
    }
  });

  return perfect_squares_array.sort(small_to_big);
}

console.log(get_squares([4, 1, 16, 1, 10, 35, 22]));

File: week7/get_squares.rb
require 'pry'

def get_squares(numbers_array)
  perfect_squares_array = []
  numbers_array.each { |num_element |
    num_element.to_i
    if(Math.sqrt(num_element)%1).zero?
      if(perfect_squares_array.include?(num_element))
      else
        perfect_squares_array << num_element
      end
    end

  }
  perfect_squares_array.sort

end

puts get_squares([4, 1, 16, 1, 10, 35, 22])
binding.pry
